addons/base_setup/models/res_users.py

```diff
@@ -12,6 +13,9 @@ class ResUsers(models.Model):
     def web_create_users(self, emails):
         emails_normalized = [tools.mail.parse_contact_from_email(email)[1] for email in emails]
 
+        if not hasattr(self, 'email_normalized'):
+            raise UserError(_("You have to install the Discuss application to use this feature."))
```

Comment:
```suggestion
            raise UserError(self.env._("You have to install the Discuss application to use this feature."))
```

Or even better, why don't you implement a fallback that just uses the e-mail field?
Something like `email_field = 'email_normalized' if 'email_normalized in self._fields else 'email'`

Reply:
Thank's for the review @kmagusiak 
I pushed the changes to use `self.env._` :+1:

> why don't you implement a fallback that just uses the e-mail field

Because it will create the user but they will not be able to login, reset a password or anything. IMO the admin should really install Discuss before inviting a user.